### PR TITLE
[Main Course] - 图数据库反欺诈实战（一）- 环境搭建

### PR DESCRIPTION
**Pull request by BulletTech2021**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (图数据库反欺诈实战（一）)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by EchoTang04** [comment]
Cypher真的很用户友好了

**Review by EchoTang04 — APPROVED** [review]
LGTM, 感谢袁老板分享！

File: docs/Main_Course/Graph_Database/2022-01-29-Neo4J-Fraud-s1.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/img/1_V/Neo4J_Sandbox.png"  />
+  <figcaption>Neo4J Sandbox</figcaption>
+</figure>
+
+点击免费使用，创建数据库，找到`Fraud Detection`即可开始探索(请记住密码)！
+
+## 3 Cypher查询语言
+
+如同SQL（Structured Query Language）用于查询关系型数据库一样，Neo4J图数据库也有自己的查询语言 - Cypher。在Cypher中，括号()用于查询节点，如`(p:Person)`，其中`p`是一个变量，`Person`是这个节点的类型.
+
```
Comment (jhqian0909) [inline docs/Main_Course/Graph_Database/2022-01-29-Neo4J-Fraud-s1.md:30]: 有一个英文句号

**Review by jhqian0909 — APPROVED** [review]

**Review by Tina0330 — APPROVED** [review]
LGTM,thanks Vincent

**Pull request merged**